SmartDeviceLink/SDLStreamingVideoLifecycleManager.m

```diff
@@ -676,23 +727,20 @@ - (void)sdl_startVideoSession {
 - (void)sdl_stopVideoSession {
     SDLLogV(@"Attempting to stop video session");
     if (!self.isStreamingSupported) {
+        SDLLogV(@"Head unit does not support video streaming. Will not send an end video service request");
         return;
     }
 
-
     if (self.isVideoConnected || self.isVideoSuspended) {
         [self.videoStreamStateMachine transitionToState:SDLVideoStreamManagerStateShuttingDown];
+    } else {
+        SDLLogV(@"No video streaming. Will not send an end video service request.");
     }
 }
 
 - (void)sdl_transitionToStoppedState:(SDLServiceType)serviceType {
```

Comment:
This method is unnecessary, just do the call at the call site

Reply:
Fixed. Removed from both audio and video managers.